test/unit/steps/sms-notify/enter-mobile/EnterMobile.test.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+const { expect } = require('test/util/chai');
+const EnterMobile = require('steps/sms-notify/enter-mobile/EnterMobile');
+const content = require('steps/sms-notify/enter-mobile/content.json');
+
+describe('EnterMobile.js', () => {
+
+    let enterMobileClass;
+
+    beforeEach(() => {
+       enterMobileClass = new EnterMobile();
+    });
+
+    afterEach(() => {
+       enterMobileClass = undefined;
```

Comment:
There is no need to set this to undefined because the next test instantiates the class EnterMobile to the same reference. If you want to clean up memory once the tests have completed you can always do the following:
`after(() => {
 enterMobileClass = undefined;
}`